### ISSUE TITLE
"Publish" vs. "Start Integration"

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis-ui#1001]
The same action is called differently in "Edit / Create integration" (Publish) and in the integration details page ("Start integration")

Plus there is a "Deploy" in the history field which is also the same.

**Comment by rhuss** [comment]
Another difference: 

* When you press "Publish" in the "Create/Edit" screen, then this action immediately.
* When you press "Start Integration" you get an extra dialog with "Are you sure ?"

**Comment by dongniwang** [comment]
@rhuss, is it possible for you to include screenshots so I can be sure that I know which step or page that you're talking about? 

When you say "the same action", are you saying the action that initializing the integration (the pod starts running)?

I think we can change "Publish" to "Deploy" in the create integration workflow for now. Not sure about the “Start Integration". 

About the confirmation message, agreed, we need to be consistent on when to ask for user confirmation...

**Comment by gashcrumb** [comment]
Yeah, `Publish` is basically create and deploy, no prompt needed.

`Save as Draft` is just save, I think a prompt is good on the detail page for `Start Integration` in this case, the user may or may not be ready to change the integration state.  Same goes with `Stop Integration`.

**Comment by gashcrumb** [comment]
Also currently `Deploy` in the history field basically updates the integration with that revision.  Probably should be a prompt on that one :-)

**Comment by gashcrumb** [comment]
Mentioned this on another issue already but yeah, as I go through the UI I see `Deploy`, `Publish`, `Start`, `Stop`, `Undeploy`, `Activated`, `Deactivated`, it feels like we're using many terms for the same thing, or possibly confusing concepts/actions.  It'd be good if we could work towards making this all more consistent.
